src/tag/item.rs

```diff
@@ -170,6 +171,9 @@ gen_map!(
 	"TDRC"                         => RecordingDate,
 	"TDOR"                         => OriginalReleaseDate,
 	"TSRC"                         => ISRC,
+	"ASIN"                         => ASIN,
```

Comment:
I have removed the last commit. This requires testing and I don't need it myself. Just thought it could be a quick and easy addition, but it apparently is more complex. That's why reviews are helpful ;)